lib/rules/require-quoted-keys-in-objects.js

```diff
@@ -44,7 +44,7 @@ module.exports.prototype = {
     check: function(file, errors) {
         file.iterateNodesByType('ObjectExpression', function(node) {
             node.properties.forEach(function(prop) {
-                if (prop.shorthand || prop.method || prop.kind !== 'init') {
+                if (prop.shorthand || prop.method || prop.kind !== 'init' || prop._babelType === 'SpreadProperty') {
```

Comment:
Hmm, we can provide function in `utils` module which could get that value, so this private property getter thing would be encapsulated and when push come to shove we could easily change it logic.